cfg.h

```diff
@@ -98,6 +98,7 @@ typedef struct {
 	char osd_lock[25];
 	uint16_t osd_lock_time;
 	char debug;
+	char pathstatus[1024] = {0};
```

Comment:
This is not data coming from the INI file, it should not be in the `cfg_t` structure.

Reply:
Fixed in this commit:

https://github.com/MiSTer-devel/Main_MiSTer/pull/887/commits/7fbc796aafba5ae27a225b5736a92c96544e73fd